### PR TITLE
add depreciation notice to ovirt-live

### PR DESCRIPTION
**Pull request by sammcj**
Changes proposed in this pull request:

- add depreciation notice

This is to reduce confusion (as I experienced) between oVirt live and oVirt node and to make people aware that oVirt live is no longer maintained.

I confirm that this pull request was submitted according to the [contribution guidelines](https://github.com/oVirt/ovirt-site/blob/master/CONTRIBUTING.md): @sammcj

### CODE REVIEW

File: source/download/ovirt-live.html.md [hunk 1]
```diff
@@ -7,6 +7,10 @@ authors: apevec, bproffitt, dneary, jvandewege, mgoldboi, obasan, sandrobonazzol
 
 # oVirt Live
 
+**Note: oVirt Live is now considered deprecated and is no longer be maintained.**
+
+Please consider the use of [oVirt Node](../../node/) instead.
```
Comment (sandrobonazzola) [inline source/download/ovirt-live.html.md]: Thanks for the patch, the deprecation notice is indeed useful. About the suggestion on using Node instead, Node is not really covering the demo functionalities provided by oVirt Live. We are working on a demo tool to replace oVirt Live

**Comment by sandrobonazzola** [comment]
CC @lveyde @gbenhaim @eedri

**Comment by sammcj** [comment]
Removed link to ovirt node

**Comment by sammcj** [comment]
Thanks for merging,

As my trial of oVirt as a replacement for XenServer continues I have a number of other areas of the documentation that I’d like to help improve, I may not get each MR correct first time but I do want to help improve the documentation.

**Pull request merged**